dcos_installer/upgrade.py

```diff
@@ -50,12 +50,20 @@
 fi
 
 SKIP_CHECKS=false
+VERBOSE=false
 
 if [[ $# -ne 0 ]]; then
-  if [[ "$1" = "--skip-checks" ]]; then
-     echo "Skipping checks"
-     SKIP_CHECKS=true
-  fi
+    for var in "$@"
+    do
```

Comment:
Let's make this `for var in "$@"; do` to be consistent with the style in the rest of this script.